src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.jpa.hibernate.dll-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/freddsdb
+spring.datasource.username=root
+spring.datasource.password=root
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
its inconvenient to use an actual db during development. 
move to h2 for that

Reply:
done